README.md

```diff
@@ -66,62 +121,44 @@ or
 rake 'handcuffs:migrate[post_restart]'
 ```
 
-You can run all migrations using
+*Note:* If you run phases out of order, or attempt to run a phase before outstanding migrations with a prerequisite phase have been run, a `HandcuffsPhaseOutOfOrderError` will be raised.
 
-```bash
-rake 'handcuffs:migrate[all]'
-```
+### Running All Migrations
 
-This differs from running `rake db:migrate` in that migrations will be run in the _order that the phases are defined in the handcuffs config_.
+In CI and local developement you may want to run all phases at one time.
 
-If you run a handcuffs rake task and any migration does not have a phase defined, an error will be raised before any migrations are run. To prevent this error, you can define a default phase for migrations that don't define one.
+Handcuffs offers a single command that will run all migrations in phases and in the configured order:
 
-```ruby
-# config/initializers/handcuffs.rb
-
-Handcuffs.configure do |config|
-  config.phases = [:pre_restart, :post_restart]
-  config.default_phase = :pre_restart
-end
+```bash
+rake 'handcuffs:migrate[all]'
 ```
 
-## Installation
+This differs from running `rake db:migrate` in that migrations will be run in batches corresponding to the _order that the phases are defined in the handcuffs config_. Again, you can use `rake handcuffs:phase_order` to preview the order ahead of time.
 
-Add this line to your application's Gemfile:
+Of course, you can always run `rake db:migrate` at any time to run all migrations using the Rails default ordering and without regard to Handcuffs phase if you wish.
 
-```ruby
-gem 'handcuffs'
-```
 
-And then execute:
 
-```bash
-bundle
-```
+## Contributing
 
-Or install it yourself as:
+Bug reports and pull requests are welcome on GitHub at <https://github.com/procore-oss/handcuffs>. This project is intended to be a safe, welcoming space for collaboration, and contributors are expected to adhere to the [Contributor Covenant](http://contributor-covenant.org) code of conduct.
 
-```bash
-gem install handcuffs
-```
 
-## Running specs
+## Running Tests Locally
```

Comment:
We should change the example command in the paragraph below this line, from
```
rake spec
```
to
```
bundle exec appraisal rspec
```
to ensure all combinations of ruby and rails are tested